docs/quickstart.rst

```diff
@@ -58,7 +58,7 @@ Finally, have Docker Compose deploy the quickstart environment:
     docker-compose up
 
 Docker Compose will set up the Inmanta server, a postgres server and two VM-like containers to experiment on.
-When Docker Compose is done deploying and the Inmanta server is running, you will be able to open the dashboard at http://127.0.0.1:8888.
+When Docker Compose is done deploying and the Inmanta server is running, you will be able to open the dashboard at http://127.0.0.1:8888/dashboard.
```

Comment:
Why keep the quickstart on the dashboard? Because of the "Using the dashboard" section?

Reply:
It's explained in the description of this PR. The web-console doesn't have support to trigger a deploy. This way the web-console cannot be used for the quickstart. This feature will be add in the future. I decided to delay the port of the quickstart until that feature is there.

Reply:
I see, thanks. I don't usually skip the description, not sure what happened.